src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -269,7 +269,7 @@ class EmojiPicker extends React.Component<IProps, IState> {
                 emojis = cat.id === "recent" ? this.recentlyUsed : DATA_BY_CATEGORY[cat.id];
             }
 
-            if (lcFilter) {
+            if (!!lcFilter) {
```

Comment:
As implied in our discussion in #element-dev: I'd still prefer `lcFilter != ""` here,  but I won't insist.

Reply:
```suggestion
            if (lcFilter !== "") {
```